django/contrib/admin/templates/admin/auth/user/unusable_password_warning.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+<template id="id_unusable_warning_template">
+  <ul id="id_unusable_warning" class="messagelist">
+    <li class="warning">
+      {% blocktranslate %}
+        The user will not be able to authenticate using a password.
+        They may still be able to authenticate using other options,
+        such as Single Sign-On or LDAP.
+      {% endblocktranslate %}
+    </li>
+    {% if form.user.has_usable_password %}
+      <li class="error">
+        {% translate "The user's current password will be lost. " %}
+      </li>
+    {% endif %}
+
```

Comment:
Could we remove this empty line?
```suggestion
```

Reply:
Whole file is gone now... 😉. It is not needed any more if CSS manages what is viewable.